### PR TITLE
fix(meetings): fix types

### PR DESCRIPTION
**Pull request by Shreyas281299**
# COMPLETES #https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-502998

## This pull request addresses

Export important classes, objects, and constants in plugin-meetings. This will make sure we dont have to import them from from '/dist' folder in WWC.

In internal-plugin-voicea, we were not creating declaration files.

## by making the following changes

- Exporting all required objects in plugin-meetings.
- Changed commands in plugin-voicea, and added tsconfig to create proper declaration files.

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [x] Tooling change
- [x] Internal code refactor

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Comment by Shreyas281299** [comment]
Code refactor of util.ts and utilv2.ts was required because with the existing code we were getting an empty declaration files.

**Comment by sreenara** [comment]
> Code refactor of util.ts and utilv2.ts was required because with the existing code we were getting an empty declaration files.

Please mark this comment next to the place where the refactor was done

File: packages/@webex/plugin-meetings/src/meeting-info/util.ts [hunk 1]
```diff
@@ -33,282 +33,290 @@ import {
...
 
-const MeetingInfoUtil: any = {};
-
-MeetingInfoUtil.extractDestination = (destination, type) => {
-  let dest = destination;
-
-  if (type === _LOCUS_ID_) {
-    if (!(destination && destination.url)) {
-      throw new ParameterError('You cannot create a meeting by locus without a locus.url defined');
+/**
```
Comment (Shreyas281299) [inline packages/@webex/plugin-meetings/src/meeting-info/util.ts:36]: The refactor of util.ts is required as without this refactor an empty declaration file was being created.

For the refactor 
1. Create a class MeetingInfoUtil
2. All the method inside the class are static. No src code changes are made.

File: packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts [hunk 2]
```diff
@@ -30,169 +30,176 @@ import ParameterError from '../common/errors/parameter';
 import LoggerProxy from '../common/logs/logger-proxy';
 import {SpaceIDDeprecatedError} from '../common/errors/webex-errors';
 
-const MeetingInfoUtil: any = {};
-const meetingInfoError =
-  'MeetingInfo is fetched with the meeting link, SIP URI, phone number, Hydra people ID, or a conversation URL.';
+/**
```
Comment (Shreyas281299) [inline packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts:33]: The refactor of utilv2.ts is required as without this refactor an empty declaration file was being created.

For the refactor 
1. Create a class MeetingInfoUtil
2. All the method inside the class are static. No src code changes are made.

File: packages/@webex/internal-plugin-voicea/package.json [hunk 3]
```diff
@@ -41,8 +42,9 @@
     "sinon": "^9.2.4"
   },
   "scripts": {
-    "build": "yarn build:src",
-    "build:src": "webex-legacy-tools build -dest \"./dist\" -src \"./src\" -js -ts -maps",
+    "ls": "pwd",
```
Comment (Neeraj-swarnkar) [inline packages/@webex/internal-plugin-voicea/package.json:45]: @Shreyas281299  why do we want to add the `ls` command here ?
Also `ls` is doing `pwd` ?  `pwd` is to get full path name of your current directory but `ls` is for listing files.

Comment (Shreyas281299) [reply packages/@webex/internal-plugin-voicea/package.json:45]: @Neeraj-swarnkar  thanks. No this was added by mistake. I was testing something thats why I added this. Removing it now.

**Review by sreenara — APPROVED** [review]
Just had one question. Changes look fine. Let's get this reviewed by the web client team as well.

File: packages/@webex/internal-plugin-voicea/package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Shreyas281299) [reply packages/@webex/internal-plugin-voicea/package.json]: No we are not using this. Ive remove this already.

File: packages/@webex/plugin-meetings/src/meeting-info/util.ts (hunk 1 above)
Comment (mkesavan13) [reply packages/@webex/plugin-meetings/src/meeting-info/util.ts:36]: Thanks for this comment. I like it :-)

**Review by mkesavan13 — COMMENTED** [review]
Just one comment. Rest looks good to me :-)

File: packages/@webex/plugin-meetings/package.json [hunk 4]
```diff
@@ -28,7 +28,7 @@
     ]
   },
   "scripts": {
-    "build": "yarn run -T tsc --declaration true --declarationDir ./dist",
+    "build": "yarn run -T tsc --declaration true --declarationDir ./dist/types",
```
[the remark this answers is not in the document]
Comment (Shreyas281299) [reply packages/@webex/plugin-meetings/package.json:31]: Yes, some changes will be required in WWC. Already raised a PR in WWC for these changes. 
https://sqbu-github.cisco.com/WebExSquared/webex-web-client/pull/5731

**Review by mkesavan13 — APPROVED** [review]

**Review by Coread — APPROVED** [review]

**Pull request merged**